### PR TITLE
[Modular] Adds some text to internals HUD button for additional support for lungs

### PR DESCRIPTION
**Pull request by nevimer**
## About The Pull Request

Did you know vox only require 10KPA, but their internals start at 16kpa? Neither did I until I was testing this.
This PR adds a new HUD element over the internals hud, it tells you information about your last breath of atmosphere. 

This does not show up for synthetics or pirates, as they do not breathe. 

## Ash Walker
dreamseeker_ufC3QoN9IW

## Vox
dreamseeker_52VPpJtUgA

https://user-images.githubusercontent.com/77420409/176815719-2c29cc32-c9d8-4999-96ad-4cc99d8a1ec8.mp4

## How This Contributes To The Skyrat Roleplay Experience

We got a lot of lung options, breathing modifiers, and theres really no way to tell your status unless you either code dived or became a robust knowledge gamer. This lets people have a bit more sensory emulation. 

## Changelog

:cl:
qol: adds a HUD for lungs
balance: You can now see your last breaths KPA 
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/lungs_hud/internals.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/mob/living/carbon/update_internals_hud_icon(internal_state)
+	. = ..()
+	internals_text_update()
+
+/mob/living/carbon/proc/internals_text_update()
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/lungs_hud/internals.dm:5]: Document this proc

File: modular_skyrat/modules/lungs_hud/internals.dm [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	var/atom/movable/screen/internals/internal_hud = hud_used?.internals
+	var/obj/item/organ/internal/lungs/mylungs = carbon.getorganslot(ORGAN_SLOT_LUNGS)
+	var/datum/gas_mixture/mix_turf = return_air()
+	var/safe_min = mylungs.safe_oxygen_min ? mylungs.safe_oxygen_min : (mylungs.safe_nitro_min ? mylungs.safe_nitro_min : (mylungs.safe_co2_min ? mylungs.safe_co2_min : (mylungs.safe_plasma_min ? mylungs.safe_plasma_min : 0)))
+	var/pressure = internal ? internal.distribute_pressure : mix_turf.return_pressure()
+	internal_hud?.maptext_y = 32
+	if(internal)
+		internal_hud?.maptext = MAPTEXT("<span style=\"font-size:5px\">LUNG:\n[safe_min]KPA\nCUR:\n[pressure]KPA\n</span>")
+	else
+		internal_hud?.maptext = MAPTEXT("<span style=\"font-size:5px\">LAST KPA:\n[pressure]\n</span>")
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/lungs_hud/internals.dm:16]: What happens if they haven't breathed in yet?

Also, you said that it doesn't display for those without lungs, but I don't see any code supporting that.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I'm still honestly not sold on this. I'd rather have a text that told me how much KPa was left in my internals, than the pressure it's being released at (because that's the information that's currently displayed in the stat panel anyway).

I'm still really not a fan of being able to know innately what your ideal intake pressure is, that should be obtainable through some different means, and is completely useless information as soon as you know it.

Furthermore, the second functionality of this seems really wrong, you shouldn't be able to tell the pressure of the room accurately by just breathing in it, that's the point of using an analyzer.

**Comment by GoldenAlpharex** [comment]
Plus, this is more like something that would be much better off taken upstream instead of being done modularly here.

**Comment by Gandalf2k15** [comment]
I'm with golden apples, a remaining KPA gaugewould be far superior

**Comment by Gandalf2k15** [comment]
re-open with changes if upstream does not accept

**Pull request closed without merging**